jaxrs-server/src/main/java/io/micronaut/jaxrs/runtime/core/JaxRsExceptionHandler.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.jaxrs.runtime.core;
+
+import io.micronaut.context.annotation.Primary;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.server.exceptions.ExceptionHandler;
+import jakarta.inject.Singleton;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+
+/**
+ * Handles JAX-RS exceptions that occur during the execution of an HTTP request.
+ *
+ * @author guillermocalvo
+ */
+@Singleton
+@Primary
```

Comment:
why the `@Primary`?

Reply:
Because `JaxRsApplicationUriNamingStrategy` was annotated as `@Primary` too. But I can remove this annotation.

Reply:
I have removed it. I have added `@Produces` which we use in other handlers.